doc: update translation of  Simple Chinese version of  readme

Matt-wzy opened the pull request:
<!--- Thank your for contributing to Reden mod! Make sure to fill out all required fields. -->
<!--- 欢迎为 Reden mod 贡献代码！请确保填写所有必填内容。 -->
<!--- IMPORTANT: If this PR addresses multiple unrelated issues, it will be closed until separated. -->
<!--- 重要：如果此 PR 解决了多个不相关的 issue，它将被关闭直到不同部分的内容被分离至多个 PR。 -->

### Description/描述

<!--- REQUIRED: Describe what changed in detail -->
<!--- 必填：详细描述了哪些代码被修改了 -->

更新 README.zh-CN.md，更新部分翻译以及添加部分新readme内容（有些实在是没看懂）

### Related Issues/相关的 issue

<!--- REQUIRED: Tag all related issues (e.g. * #123) -->
<!--- If this PR fixes a bug, please specify (e.g. * fix #123) -->
<!--- If this PR adds a feature, please specify (e.g. * resolve #123) --> 
<!--- If this PR addresses multiple issues, these issues must be related to one other -->
<!--- 必填：标记所有相关的 issue -->
<!--- 如果此 PR 修复了一个 bug，请标记（例如 * fix #123） -->
<!--- 如果此 PR 添加了一个新功能，请标记（例如 * resolve #123） -->
<!--- 如果此 PR 解决了多个 issue，这些 issue 必须是相关的 -->

* fix NA
* resolve NA

### Checklist/清单

<!--- Add things that are not yet implemented above -->
<!--- 将仍未实现的内容添加至上方 -->

<!--- Check the following items by changing [ ] to [x] -->
<!--- 将 [ ] 改为 [x] 来勾选下方的内容 -->
- [ ] The issues this PR addresses are related to each other/此 PR 解决的 issue 是相关的
- [ ] My changes are all related to the related issue above/我的修改都与上方的相关 issue 有关
- [x] My changes generate no new warnings/我的修改没有产生新的警告
- [ ] My code builds and runs on my machine/我的代码在我的电脑上可以编译和运行
- [ ] I documented my code/我已经为我的代码添加了注释/文档

### Screenshots

<!--- REQUIRED: if issue is UI related -->
<!--- 必填：如果 issue 与 UI 有关 -->

<!--- IMPORTANT: Fill out all required fields. Otherwise we might close this PR temporarily -->
<!--- 重要：请填写所有必填内容。否则我们可能会暂时关闭此 PR -->
<!--- Make sure to mark the PR as draft if you are still working on it, and mark it as "ready for review" when you are done -->
<!--- 如果你仍未完成此 PR，请将其标记为草稿（GitHub PR 中的 draft 功能）。当你完成时，请点击 "ready for review" -->

Matt-wzy committed:
doc: update translation of  Simple Chinese version of  reademe

zly2006 commented on README.zh-CN.md:
```diff
@@ -1,45 +1,55 @@
-# Reden 是我们创造的
+# Reden -- 在MC中的撤销(Ctrl +Z) 与重做(Ctrl+Y) 模组
```
请您先仔细看看介绍，谁说这个模组的功能**主要是**undo了？

Matt-wzy committed:
doc: fix a misleading title

Qubik65536 commented on README.zh-CN.md:
```diff
@@ -1,45 +1,55 @@
 
-## 倒退刻
-> 早期访问阶段
+快捷键 (malilib -> General)	默认
```
需要与英文 README 同步，使用表格

Matt-wzy replied:
此处是一个低级失误，已在最新的提交中修正

Qubik65536 commented on README.zh-CN.md:
```diff
@@ -1,45 +1,55 @@
+## 撤销&重做
 
-Ctrl+Z 可以立即撤销之前的操作，使得开发红石机器更加方便！Ctrl+Y可以重做。
+Ctrl+Z  撤销 ，妈妈再也不怕我炸珍珠炮啦！Ctrl+Y 重做（取消撤销），爸爸再也不怕我把世界撤销到初始状态啦！
```
```suggestion
Ctrl+Z 撤销 ，妈妈再也不怕我炸珍珠炮啦！Ctrl+Y 重做（取消撤销），爸爸再也不怕我把世界撤销到初始状态啦！
```

另外，我认为最好遵循英文原文内容（待讨论）

Matt-wzy replied:
已开启 Allow edits by maintainers 讨论完毕后可以直接修改此处，暂时保留

Qubik65536 commented on README.zh-CN.md:
```diff
@@ -1,45 +1,55 @@
 
-Ctrl+Z 可以立即撤销之前的操作，使得开发红石机器更加方便！Ctrl+Y可以重做。
+Ctrl+Z  撤销 ，妈妈再也不怕我炸珍珠炮啦！Ctrl+Y 重做（取消撤销），爸爸再也不怕我把世界撤销到初始状态啦！
 更多信息请参阅[wiki页面](https://wiki.redenmc.com/Undo-and-Redo)
```
```suggestion
更多信息请参阅 [wiki页面](https://wiki.redenmc.com/Undo-and-Redo)
```

Markdown 格式要求，以后会在单独的文档中写明，目前直接更改

Matt-wzy replied:
已在最新的提交中修正

Qubik65536 commented on README.zh-CN.md:
```diff
@@ -1,45 +1,55 @@
 > 开发中
 
-RVC，红石版本控制，机器历史和差异分析
+红石版本控制 RVC( Redstone Version Control)，管理机器的修改历史、分析历史记录。
```
``` suggestion
红石版本控制 RVC (Redstone Version Control)，管理机器的修改历史、分析历史记录。
```

Markdown 格式问题，上面那个可以是项目标准没公布，这个错的离谱了

Matt-wzy replied:
已在最新的提交中修正

Matt-wzy committed:
doc: fixed basic Markdown syntax issues in the README.zh-CN document

Qubik65536 commented on README.zh-CN.md:
```diff
@@ -1,45 +1,59 @@
 
-Ctrl+Z 可以立即撤销之前的操作，使得开发红石机器更加方便！Ctrl+Y可以重做。
-更多信息请参阅[wiki页面](https://wiki.redenmc.com/Undo-and-Redo)
+Ctrl+Z 撤销 ，妈妈再也不怕我炸珍珠炮啦！Ctrl+Y 重做（取消撤销），爸爸再也不怕我把世界撤销到初始状态啦！
```
请尊重原文

Matt-wzy committed:
doc: strictly adhere to the original text mentioned in the link: https://github.com/zly2006/reden-is-what-we-made/pull/77#discussion_r1439703248

https://github.com/zly2006/reden-is-what-we-made/pull/77#discussion_r1439703248

Qubik65536 committed:
style(README.zh-CN)

Qubik65536 commented:
把之前改对的格式化改回去了真是无语了
